app/components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,100 @@
+import MTextField from "@mui/material/TextField";
+import MInputAdornment from "@mui/material/InputAdornment";
+import { ComponentPropsWithoutRef, forwardRef } from "react";
+import MuiIcons from "../MuiIcons";
+import { MuiIconsProps } from "../MuiIcons/MuiIcons";
+
+interface ITextInputProps extends ComponentPropsWithoutRef<"input"> {
```

Comment:
can we import the props interface aswell?

Reply:
done